doc/PROTOCOL-HTTP2.md

```diff
@@ -106,10 +106,11 @@ For requests, **EOS** (end-of-stream) is indicated by the presence of the END_ST
 * **Response** → (Response-Headers \*Length-Prefixed-Message Trailers) / Trailers-Only
 * **Response-Headers** → HTTP-Status [Message-Encoding] [Message-Accept-Encoding] Content-Type \*Custom-Metadata
 * **Trailers-Only** → HTTP-Status Content-Type Trailers
-* **Trailers** → Status [Status-Message] \*Custom-Metadata
+* **Trailers** → Status [Status-Message] [Status-Details] \*Custom-Metadata
 * **HTTP-Status** → ":status 200"
 * **Status** → "grpc-status" 1\*DIGIT ; 0-9
 * **Status-Message** → "grpc-message" Percent-Encoded
+* **Status-Details** → "grpc-status-details-bin" {base64 encoded value} # See notes below.
```

Comment:
I think the comment character here is a `;`, not a `#`.

Reply:
There are 4 existing uses of `#` for a comment.  Do you want me to change them all?

Reply:
Might as well be consistent, yeah.

Reply:
Done.  There's one more `#` comment, but it's in a code block and not the ABNF, so I left it.